board.py

```diff
@@ -9,6 +9,7 @@
 boards = {
     "Adafruit Feather M0 RFM69 with samd21g18": "feather_m0_rfm69",
     "Adafruit PyPortal Titano with samd51j20": "pyportal_titano",
+    "Adafruit PyPortal with samd51j20": "pyportal",
```

Comment:
Not related to Linux, but I'll allow it

Reply:
whoops, that one slipped in from when I tested the update command.